docker/assertj/src/main/java/com/github/dockerjava/assertions/DockerJavaAssert.java

```diff
@@ -31,6 +35,12 @@ public ContainerAssert container(String name) {
       return new ContainerAssert(container);
    }
 
+   public CubeOutputAssert withContainer(String name) {
+         String[] cmd = {"bash", "-c", "for i in $(ps -axo pid --no-headers); do echo `ps -p $i -o comm=`; done"};
```

Comment:
How about using this method https://github.com/arquillian/arquillian-cube/blob/master/docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java#L813 it is a pure API solution and we avoid two things, the first is having to do 2 REST commands and the second one and more important, not all containers has installed `bash`. So we avoid being incompatible with these ones.